fluentd/Dockerfile.centos7

```diff
@@ -18,6 +18,8 @@ RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
     yum install -y --setopt=tsflags=nodocs \
       make \
       bc \
+      git \
+      libffi-devel \
```

Comment:
We should probably convert this to a 2 stage Dockerfile so we don't install the build tools.  Maybe a followon PR

Reply:
ok